sql/src/test/java/io/crate/integrationtests/SysNodeCheckerIntegrationTest.java

```diff
@@ -78,4 +107,20 @@ public void testUpdateAcknowledgedFromReference() throws Exception {
         execute("update sys.node_checks set acknowledged = not passed where passed = false");
         assertThat(response.rowCount(), is(rc));
     }
+
+    /**
+     * Check if cluster name check is raising a warning, when path.data
+     * contains cluster name folder.
+     *
+     * TODO: Remove this when ES >= 6.0
+     */
+    @Test
+    public void testClusterNameInPathDataNodesSysCheck() throws Exception {
+        execute("select description, passed from sys.node_checks where not passed and id=1000");
+        assertThat(response.rowCount(), is(2L));
+        String description = (String) response.rows()[0][0];
+        assertTrue(description.contains(dirs.get(0)) || description.contains(dirs.get(1)));
```

Comment:
We generally prefer the `assertThat(..., is()` pattern. Or in this case `assertThat(description, contains(..))`  - this will result in better error messages.